app/assets/stylesheets/components/comments.scss

```diff
@@ -46,6 +36,12 @@
       width: auto;
       object-fit: contain;
     }
+
+    pre:not(.highlight),
+    div.highlight {
```

Comment:
Is the `div` as part of the selector necessary?

Reply:
Yeap, since `.highlight` is a class name not only for DIVs but also for PREs.. that's why i have this non-standard selector a line above with `:not...`